8Knot/pages/contributions/visualizations/issues_over_time.py

```diff
@@ -100,18 +100,21 @@
                             justify="start",
                         ),
                         dbc.Row(
-                            dcc.DatePickerRange(
-                                id=f"date-picker-range-{PAGE}-{VIZ_ID}",
-                                min_date_allowed=dt.date(2005, 1, 1),
-                                max_date_allowed=dt.date.today(),
-                                initial_visible_month=dt.date(dt.date.today().year, 1, 1),
-                                clearable=True,
-                                start_date=dt.date(
-                                    dt.date.today().year - 2,
-                                    dt.date.today().month,
-                                    dt.date.today().day,
+                            dbc.Col(
+                                dcc.DatePickerRange(
+                                    id=f"date-picker-range-{PAGE}-{VIZ_ID}",
+                                    min_date_allowed=dt.date(2005, 1, 1),
+                                    max_date_allowed=dt.date.today(),
+                                    initial_visible_month=dt.date(dt.date.today().year, 1, 1),
+                                    clearable=True,
+                                    start_date=dt.date(
+                                        dt.date.today().year - 2,
+                                        dt.date.today().month,
+                                        dt.date.today().day,
+                                    ),
+                                    className="dark-date-picker",
                                 ),
-                                className="dark-date-picker",
```

Comment:
Can you make this a 1 line change as it is for all of the initial files?

Reply:
there were ~2 visualizations where I could not do this with a one line change because they were not already wrapped in a `dbc.Col`. I tried to add a `width` parameter to the existing `dbc.Row` but this caused the app to crash and not start up because `width` is not a valid argument for `Row`. these multiline changes are necessary (all they do is wrap the picker in a Col)